src/store-api/src/logstore.rs

```diff
@@ -65,6 +69,11 @@ pub trait LogStore: Send + Sync + 'static + std::fmt::Debug {
     /// Create a namespace of the associate Namespace type
     // TODO(sunng87): confusion with `create_namespace`
     fn namespace(&self, id: namespace::Id) -> Self::Namespace;
+
+    /// Mark the entry id of given namespace as stable so that logstore can safely delete
```

Comment:
Denote the `id` is inclusive?
```suggestion
    /// Mark the entry ids `<= id` of given `namespace` as stable so that logstore can safely delete
```